node-tests/addon-test.js

```diff
@@ -216,6 +216,112 @@ describe('ember-cli-babel', function() {
       }));
     });
 
+    describe("Opting out of the ember modules API polyfill", function () {
+      let dependencies;
+
+      beforeEach(function () {
+        dependencies = {};
+        let project = {
+          root: input.path(),
+          emberCLIVersion: () => "2.16.2",
+          dependencies() {
+            return dependencies;
+          },
+          addons: [],
+          targets: {
+            browsers: ["ie 11"],
+          },
+        };
+
+        this.addon = new Addon({
+          project,
+          parent: project,
+          ui: this.ui,
+        });
+
+        project.addons.push(this.addon);
+      });
+
+      function buildEmberSourceFixture(version) {
+        return {
+          node_modules: {
+            "ember-source": {
+              "package.json": JSON.stringify({ name: "ember-source", version }),
+              "index.js": "module.exports = {};",
+            },
+          },
+        };
+      }
+
+      it(
+        "should replace imports with Ember Globals",
+        co.wrap(function* () {
+          const PRE_GLOBAL_RESOLVER_DEPRECATION_VERSION = "3.26.0";
+          dependencies[
+            "ember-source"
+          ] = PRE_GLOBAL_RESOLVER_DEPRECATION_VERSION;
+          input.write(
+            buildEmberSourceFixture(PRE_GLOBAL_RESOLVER_DEPRECATION_VERSION)
+          );
```

Comment:
Why is this variable named `PRE_GLOBAL_RESOLVER_DEPRECATION_VERSION`?

Reply:
AHA! I think you mean "global resolver" as in transforming to `Ember.Foo`. Unfortunately, we actually already have a thing that is called "globals resolver" and it is in fact deprecated too. 

Sorry for the annoyance, but lets find a different name that doesn't cause future rwjblue confusion.

Reply:
Does `PRE_EMBER_MODULE_IMPORTS_VERSION` sound better to you?

Reply:
Ya, sounds great. Apologies again for the bike shedding.